Content.Server/Tools/ToolSystem.Welder.cs

```diff
@@ -49,7 +49,7 @@ public void InitializeWelders()
         private void OnGetMeleeDamage(EntityUid uid, WelderComponent component, ref GetMeleeDamageEvent args)
         {
             if (component.Lit)
-                args.Damage += component.LitMeleeDamageBonus;
+                args.Damage = component.LitDamage;
         }
```

Comment:
This will work, if it comes last in the event subscription chain, which means it will conflict if there's ever any other system that interacts with damage for these items.

Reply:
Okay I now have a doubt. The difference between Specific and Relative lit damage is that, of course, code like relative damage can be applied more than once. So if ever in the future something ELSE tries to work like this, it will work with Relative, but not Specific. But... wer are talking about the BASE damage of the weapon here. 
Is a scenario where multiple systems are trying to change that plausible? Or is this just a moment where I am adding tech debt out of good intentions and lack of foresight

Reply:
Right, so this is what I was trying to explain before. Due to the nature of how this event works, it should be caught as early as possible, in the event of other components wanting to add (or subtract) damage. `SubscribeLocalEvent` has a `before` field that you can add types to in order to make this subscription come earlier. The idea is you set the damage, then other event handlers can change it as it goes forward.

Here's an example of what I mean:
https://github.com/space-wizards/space-station-14/blob/fff7fda676eec723bae63509c65e6c57329b9fd4/Content.Server/Crayon/CrayonSystem.cs#L35-L36

The issue will be that this has to be updated any time someone wants to add another handler that could affect this later on, so in that case, it might be better for those writing those systems to use an `after:` based subscription instead.

It's one of those things that can go either way, and I'm not immediately sure what the best solution is.